### DISCUSSION TITLE
The size after building or publishing the app is too large.

### DISCUSSION THREAD

**Discussion by yuya-tani** [microsoft/react-native-windows#10218] [Q&A] [answered]
Hi.

I installed it according to the tutorial and
I created an app package for publishing.

Then, before I knew it, the size of the project
It was inflated to 19 GB.

Root "node_modules": 11 GB
\ windows \ {ProjectName}: 5 GB
\ windows \ x64: 1GB
Others ...

Isn't the size too big?
If it becomes this size in a project that has been slightly modified from the start
It will be difficult to create many projects.

Do you guys have this size? Is this normal?
Is there a way to solve it?

  Best regard.

**Comment by chrisglein** [comment, marked as answer]
You can try using the [nuget binary](https://microsoft.github.io/react-native-windows/docs/NuGet) so you don't have to build all that (saving on time and storage).
